src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -194,7 +194,7 @@ static int32_t cc_rides(InteractiveConsole& console, const arguments_t& argv)
                 }
                 else
                 {
-                    int32_t res = set_operating_setting(ride_index, RideSetSetting::RideType, type);
+                    int32_t res = set_operating_setting(static_cast<ride_id_t>(ride_index), RideSetSetting::RideType, type);
```

Comment:
you could do the cast on L185 to save a small amount of changes.

Reply:
Theres no saving here, L191 needs changing then, its a local scope also.